packages/eas-cli/src/credentials/ios/api/GraphqlClient.ts

```diff
@@ -204,8 +205,15 @@ export async function createOrGetExistingAppleAppIdentifierAsync(
     if (!appleTeam) {
       throw new AppleTeamMissingError();
     }
+    const parentAppleAppIdentifier = parentBundleIdentifier
+      ? await AppleAppIdentifierQuery.byBundleIdentifierAsync(account.name, parentBundleIdentifier)
```

Comment:
is it safe to assume the parentAppleAppIdentifier exists? if not, better to call createOrGetExistingAppleAppIdentifierAsync
```suggestion
      ? await createOrGetExistingAppleAppIdentifierAsync({account, bundleIdentifier: parentBundleIdentifier}...)
```